### ISSUE TITLE
underscore prefixed names not recognized #48

### ISSUE THREAD

**Issue by glodov** [pug-php/pug#144]
It appears in version ^2.6.

I want to use __page, when I use page it does work.
I tried $__page, does not help.

```pug
head
  title= __page.seo.title
  meta(charset='utf-8')
```

But it does work with __config.

```pug
      dl
        each value, key in __config
```

Do you have some predefined __ variables?

**Comment by kylekatarnls** [comment]
It will be far different in the next version but until 3.0, the expressions after `=` and in `each`, `if`, `while`, etc. are not handle in the same way. For now, if you have `_` prefixed variables, you will need to write pure PHP to handle them with no risks:
```pug
// array
title=$__page['seo']['title']
// object
title=$__page->seo->title
```

**Comment by glodov** [comment]
Thanks, I just stick to non prefixed variables and it works ;)

**Comment by kylekatarnls** [comment]
I close this since it already works in https://github.com/phug-php/phug (our next engine), so it will be released on 3.0.0.
